.eslintrc.js

```diff
@@ -12,14 +11,11 @@ const options = {
 		'plugin:node/recommended',
 		'plugin:react/recommended',
 		'plugin:import/recommended',
-		'plugin:@typescript-eslint/recommended'
+		'plugin:@typescript-eslint/recommended',
+		'plugin:prettier/recommended'
```

Comment:
Hi, @akashgp09 Do we require this line "plugin:prettier/recommended" to work prettier with eslint ?
Try removing this line to get rid of lint error on checks.

Reply:
I read somewhere that it enables the main config from eslint-config-prettier, which turns off some ESLint core rules that conflict with Prettier.

Reply:
also the reason the link check is failing is because it cannot find **eslint-plugin-prettier** dependency.

Reply:
@MonkeyDo should we remove the line from the eslint file as suggested by @NNOVATIVEGAMER or is there any work around to fix it

Reply:
> I imagine that it enables the main config from eslint-config-prettier, which turns off some ESLint core rules that conflict with Prettier. seen to be used commonly

Yes , you are right. Have you tried reinstalling the "eslint-plugin-prettier" package.

Reply:
Yes I did. And the linter isn't throwing any error locally. I think there is some issue with the lint Runner workflow.